Assets/Scripts/Models/Furniture.cs

```diff
@@ -701,43 +843,75 @@ public void Deconstruct()
         // should get garbage-collected.
     }
 
+    /// <summary>
+    /// Gets the tile that is used to do a job.
+    /// </summary>
+    /// <returns>Tile that is used for jobs</returns>
     public Tile GetJobSpotTile()
     {
         return World.Current.GetTileAt(Tile.X + (int)JobSpotOffset.x, Tile.Y + (int)JobSpotOffset.y, Tile.Z);
     }
 
+    /// <summary>
+    /// Gets the tile that is used to spawn new objects or spacemen
```

Comment:
I would go with "objects (i.e. Inventory, Character)", since in the future Mods can use it for anything and we don't what it will be used for.